examples/README.md

```diff
@@ -1,3 +1,12 @@
+# Running the examples
+
+(The follow assumes you have checked out the complete puppeteer repository, and installed puppeteer's dependencies via `yarn`.)
```

Comment:
How about this:

Assuming you have a check out of this repo and have run `yarn` (or `npm i`) to install the dependencies, the examples can be run from the root folder like so:

```sh
NODE_PATH=../ node examples/search.js
```

Reply:
Sure, changed! (Small tweaks to wording.)